test/src/edu/stanford/nlp/process/StemmerTest.java

```diff
@@ -0,0 +1,188 @@
+package edu.stanford.nlp.process;
```

Comment:
To possibly save myself the hassle of checking myself, let me ask - have you compared these results to the results from a reference implementation such as the ones provided here?  https://tartarus.org/martin/PorterStemmer/

Reply:
I updated this: Now I test it against the official implementation of Martin Porters algorithm by himself, as done in commit [f35090a](https://github.com/stanfordnlp/CoreNLP/pull/1036/commits/f35090ae990699a477ad967c05a34aa98d560e2f).